lib/addons/apache-airflow/index.ts

```diff
@@ -0,0 +1,302 @@
+import { HelmAddOn } from '../helm-addon/index';
+import { AwsLoadBalancerControllerAddOn } from "../aws-loadbalancer-controller";
+import { EfsCsiDriverAddOn } from "../efs-csi-driver";
+
+import { ClusterInfo } from '../../spi/types';
+import { Values } from "../../spi";
+import { setPath, createNamespace, createServiceAccount } from "../../utils";
+import { IFileSystem } from "aws-cdk-lib/aws-efs";
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface AirflowAddOnProps extends HelmAddOnUserProps {
+    /**
+     * Enable Load Balancer
+     */
+    enableAlb?: boolean,
+
+    /**
+     * Name of the {@link certificateResourceName} to be used for certificate look up. 
+     * @see {@link ImportCertificateProvider} and {@link CreateCertificateProvider} for examples of certificate providers.
+     */
+    certificateResourceName?: string,
+
+    /**
+     * Names of the S3 Bucket provider named resources (@see CreateS3BucketProvider, @see ImportS3BucketProvider).
+     * S3 Bucket provider is registered as named resource providers with the EksBlueprintProps.
+     */
+    s3Bucket?: string,
+
+    /**
+     * Enable EFS for persistent storage of DAGs
+     */
+    enableEfs?: boolean,
+
+    /**
+     * Names of the EFS File System provider named resources (@see CreateEfsFileSystemProvider, @see LookupEfsFileSystemProvider).
+     * EFS File System provider is registered as named resource providers with the EksBlueprintProps.
+     * This is required if EFS is enabled
+     */
+    efsFileSystem?: string,
+
+    /**
+     * TODO: Enable RDS
+     */
+    // enableRds?: boolean,
+
+    /**
+     * TODO: DB Configurations for RDS instance access - must have RDS enabled
+     */
+    // dbConfig?: {
+    //     username: string,
+    //     password: string,
+    //     dbName: string 
+    // }
+    
+    /**
+     * Enable Logging with S3
+     */
+    enableLogging?: boolean,
+}
+
+const AIRFLOW = 'airflow';
+const RELEASE = 'blueprints-addon-apache-airflow';
+
+/**
+ * Default props to be used when creating the Helm chart
+ */
+ const defaultProps: HelmAddOnProps = {
+    name: AIRFLOW,
+    namespace: AIRFLOW,
+    chart: AIRFLOW,
+    version: "1.9.0",
+    release: RELEASE,
+    repository:  "https://airflow.apache.org",
+    values: {}
+};
+
+/**
+ * This add-on is currently not supported. It will apply the latest falco helm chart but the latest AMI does not have stock driver supported and
+ * driver build in the init fails atm. 
+ */
+export class ApacheAirflowAddOn extends HelmAddOn {
+
+    readonly options: AirflowAddOnProps;
+
+    constructor(props?: AirflowAddOnProps) {
+        super({...defaultProps, ...props});
+        this.options = this.props as AirflowAddOnProps;
+    }
+
+    deploy(clusterInfo: ClusterInfo): Promise<Construct> {
+        const cluster = clusterInfo.cluster;
+
+        // Create Namespace
+        const ns = createNamespace(this.options.namespace!, cluster, true, true);
```

Comment:
is there a case when they want to install into an existing namespace? we can call out in the documentation. Generally, we also add a `createNamespace` attribute